### PR TITLE
Vet-Clinic-Database

### PR DESCRIPTION
**Pull request by AhmadiGu**
Screenshot (408)
Screenshot (407)
Screenshot (406)
Screenshot (405)
Screenshot (404)
Screenshot (403)
Screenshot (402) 
Screenshot (409) 
Screenshot (400)

**note:** in each image you can see the queries on the left and the result on the right.
### Milestone: Create a database
*The following changes are being introduced*

- create the database
- create the animal table
- insert data into the table
- show the data according to some conditions

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered IS true;
+
+SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (Nemwel-Boniface) [inline queries.sql:16]: - [ ] As it is a strict requirement that you include screenshots of your query results in your pull request, kindly ensure that you include them.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @AhmadiGu      :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done a great job so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (AhmadiGu) [reply queries.sql:16]: Hi, 
okay, I will add it.
 thanks

**Review by prolajumokeoni — APPROVED** [review]
Hi @AhmadiGu 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**